can/interfaces/pcan/pcan.py

```diff
@@ -16,9 +14,8 @@
 from can.util import len2dlc, dlc2len
 from .basic import *
 
-boottimeEpoch = 0
 try:
-    import uptime
+    import uptime # isn't this from a library?
```

Comment:
Correct. Keep this as before. It will produce absolute timestamps if uptime is installed.